### PR TITLE
Add table of content to contests report

### PR DESCRIPTION
**Pull request by Simon-Busch**
- [x] Add relevant plugin
- [x] Update config
- [x] Update component
- [x] Render TOC
- [x] Style TOC

### CODE REVIEW

File: gatsby-config.js [hunk 1]
```diff
@@ -61,6 +61,18 @@ module.exports = {
...
+          `gatsby-remark-autolink-headers`,
+          {
+            resolve: `gatsby-remark-table-of-contents`,
+            options: {
+              exclude: "Table of Contents",
+              tight: false,
+              ordered: false,
+              fromHeading: 1,
+              toHeading: 6,
+              className: "table-of-contents"
```
Comment (captainmangoC4) [inline gatsby-config.js:73]: I'd consider setting `elements` to just `h1`. Maybe `h2` as well if you think it's important to have links to the specific findings as well as the risk sections, but I would definitely exclude `h3` and above.

File: src/pages/reports/{MarkdownRemark.fields__slug}.js [hunk 2]
```diff
@@ -31,10 +31,14 @@ function ReportPageTemplate({ data }) {
             </h1>
             <h4>{page.frontmatter.date}</h4>
           </div>
-          <div
-            className="report-contents"
-            dangerouslySetInnerHTML={{ __html: DOMPurify.sanitize(page.html) }}
-          />
+          <div className="report-container">
+            <h2>Table of content</h2>
+            <div className="report-toc" dangerouslySetInnerHTML={{ __html: DOMPurify.sanitize(page.tableOfContents) }}/>
```
Comment (captainmangoC4) [inline src/pages/reports/{MarkdownRemark.fields__slug}.js:36]: Can you update the layout of the page to have the table of contents on to the side and statically positioned?

**Review by captainmangoC4 — CHANGES_REQUESTED** [review]
Great job and nice find! I didn't know there was a plugin for this!

File: src/pages/reports/{MarkdownRemark.fields__slug}.js [hunk 3]
```diff
@@ -31,12 +40,17 @@ function ReportPageTemplate({ data }) {
             </h1>
             <h4>{page.frontmatter.date}</h4>
           </div>
-          <div
-            className="report-contents"
-            dangerouslySetInnerHTML={{ __html: DOMPurify.sanitize(page.html) }}
-          />
+          <div className="report-container">
+            <h2>Table of content</h2>
```
Comment (joshuashort) [inline src/pages/reports/{MarkdownRemark.fields__slug}.js:44]:
```suggestion
            <h2>Table of contents</h2>
```

**Review by joshuashort — APPROVED** [review]

File: src/pages/reports/{MarkdownRemark.fields__slug}.js (hunk 2 above)
Comment (Simon-Busch) [reply src/pages/reports/{MarkdownRemark.fields__slug}.js:45]: @captainmangoC4 After our discussion, I think that this is outdated right as I added the button to back to stop :) 

Merging is blocked :-)

**Comment by Simon-Busch** [comment]
@captainmangoC4 can you re-check the PR and eventually approve if OK for you ? :)

File: src/pages/reports/{MarkdownRemark.fields__slug}.js (hunk 2 above)
Comment (captainmangoC4) [reply src/pages/reports/{MarkdownRemark.fields__slug}.js:45]: Yeah, the back to top button is a nice solution.

**Review by captainmangoC4 — APPROVED** [review]

**Pull request merged**